Description: Changes default browser from Opera (which is non-free) to Epiphany.
Author: Marek Buras <cyfr0n@go2.pl>
Origin: vendor
Forwarded: no
Last-Update: 2013-01-29
--- a/src/common/xchat.c
+++ b/src/common/xchat.c
@@ -604,7 +604,7 @@
 	"NAME WII\n"			"CMD quote WHOIS %2 %2\n\n";
 
 static char defaultconf_urlhandlers[] =
-		"NAME Open Link in Opera\n"		"CMD !opera -remote 'openURL(%s)'\n\n";
+		"NAME Open Link in the default browser\n"	"CMD !x-www-browser '%s'\n\n";
 
 #ifdef USE_SIGACTION
 /* Close and open log files on SIGUSR1. Usefull for log rotating */
